metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/common/BaseStellarProcessor.java

```diff
@@ -143,7 +143,11 @@ public T parse(final String rule, final VariableResolver variableResolver, final
     try {
       return clazz.cast(expression
           .apply(new StellarCompiler.ExpressionState(context, functionResolver, variableResolver)));
-    }finally {
+    }
```

Comment:
Maybe we can have stellar exception builders that provide standard, consistent exception building

Reply:
Yeah, agreed.